assembl/templates/sentry.jinja2

```diff
@@ -0,0 +1,5 @@
+<script src="https://browser.sentry-cdn.com/4.2.4/bundle.min.js" crossorigin="anonymous"></script>
```

Comment:
Hmm, can we do better? We're breaking centrality of package definitions by doing this (and not in yarn/package.json. On other hand, putting it on yarn won't work with our future CI/CD approach 😢. Any other suggestions?

Reply:
I agree. I don't know how I can install it and import it in the jinja template. I tried but I didn't want to spend to much time on it. If you have an idea, I'll do the change with pleasure :-)

Reply:
Sigh, only thing I can think of, which is a huge hassle, is to put this into package.json/yarn.lock, and import it into Assembl and introduce code splitting in webpack. I'm not even sure this would work, as the DOM will be making JS calls on an object that might not yet be loaded. The bundle.min.js on my machine is 91K.

I'm not certain there is any easier way to load this, unless we create an API endpoint to pass the sentry DSN/secret through before react init, and not use the template provided by Pyramid.

Reply:
if the sentry dsn is not sensitive data, I could put it in a data attribute in the DOM and retrieve in it React code (to avoid to create an API endpoint for this but even with the API endpoint, this data is exposed).